contracts/Governance/Governor.sol

```diff
@@ -170,7 +168,7 @@ contract Governor {
         string memory description
     ) public returns (uint256) {
         require(
-            token.getPriorVotes(msg.sender, sub256(block.number, 1)) > proposalThreshold(),
+            token.getPriorVotes(msg.sender, sub256(block.number, 1)) >= proposalThreshold(),
```

Comment:
Notice this logic change: `>` changed to `>=`. Please double check this makes sense.

Reply:
It does, `>=` = can propose, `<` = can't propose, existing proposals are moot